cmd/discover/main_test.go

```diff
@@ -27,5 +27,5 @@ func TestMissingArguments(t *testing.T) {
 	process, err := Start(cmd, nil, nil)
 	gt.Expect(err).NotTo(HaveOccurred())
 	gt.Eventually(process, 5*time.Second).Should(Exit(1))
-	gt.Expect(process.Err).To(gbytes.Say("empty string that is mandatory"))
+	gt.Expect(process.Err).To(gbytes.Say("IdentityPath is mandatory and cannot be empty"))
```

Comment:
Hi @yeasy
In the Go v1 series, when looping through a map with a range, the order in which items are retrieved is randomized by design. 
https://go.dev/doc/go1#iteration

Therefore, it is necessary to update the expected values for the tests accordingly to accommodate both cases, or make modifications to identify the element (e.g., setting only one missing flag).
```suggestion
	gt.Expect(process.Err).To(gbytes.Say("IdentityPath|KeyPath is mandatory and cannot be empty"))
```

Reply:
Updated, thanks!